linux_os/guide/system/auditing/policy_rules/audit_basic_configuration/rule.yml

```diff
@@ -0,0 +1,59 @@
+documentation_complete: true
+
+prodtype: rhel8
+
+title: 'Configure basic parameters of Audit system'
+
+description: |-
+    Perform basic configuration of Audit system.
+    Make sure that any previously defined rules are cleared, the auditing system is configured to handle sudden bursts of events, and in cases of failure, messages are configured to be directed to Syslog.
+
+    The following rules configure audit as described above:
+    <pre>
```

Comment:
There are indentation/quote matching issues in the file - I recommend to use an [online YAML parser](https://yaml-online-parser.appspot.com/) to make sure that the result meets expectations.

Reply:
I used yamllint to find the redundant apostrophe. Now it compiles. Does it have some wrong indentation now?